app_dart/lib/src/request_handlers/flaky_handler_utils.dart

```diff
@@ -15,8 +15,7 @@ import '../service/github_service.dart';
 import '../../protos.dart' as pb;
 
 // String constants.
-const String kTeamFlakeLabel = 'team: flakes';
-const String kSevereFlakeLabel = 'severe: flake';
+const String kFlakeLabel = 'c: flake';
 const String kFrameworkLabel = 'framework';
 const String kToolLabel = 'tool';
 const String kEngineLabel = 'engine';
```

Comment:
shall all these team labels be mapped to `team-*` according to https://github.com/flutter/flutter/wiki/Triage#new-team-issue-triage-process?